zeppelin-splicemachine/pom.xml

```diff
@@ -0,0 +1,126 @@
+  ~ If not, see <http://www.gnu.org/licenses/>.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>zeppelin-splicemachine-0.8.0</artifactId>
+    <description>Splice Machine Zeppelin interpreter</description>
+    <parent>
+        <artifactId>spliceengine-parent</artifactId>
+        <groupId>com.splicemachine</groupId>
+        <version>2.8.0.1845-SNAPSHOT</version>
+    </parent>
+    
+    <properties>
+        <interpreter.name>splicemachine</interpreter.name>
+        <zeppelin.version>0.8.0</zeppelin.version>
+        <!--test library versions-->
+        <mockrunner.jdbc.version>1.0.8</mockrunner.jdbc.version>
+        <commons.dbcp2.version>2.0.1</commons.dbcp2.version>
+        <commons.pool2.version>2.3</commons.pool2.version>
+        <commons.logging.version>1.1.1</commons.logging.version>
+        <slf.logging.version>1.7.10</slf.logging.version>
+        <log4j.version>2.11.1</log4j.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.zeppelin</groupId>
+            <artifactId>zeppelin-interpreter</artifactId>
+            <version>${zeppelin.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.zeppelin</groupId>
+            <artifactId>zeppelin-jdbc</artifactId>
+            <version>${zeppelin.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.splicemachine</groupId>
+            <artifactId>db-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.mockrunner</groupId>
+            <artifactId>mockrunner-jdbc</artifactId>
+            <version>${mockrunner.jdbc.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <version>${commons.dbcp2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-pool2</artifactId>
+            <version>${commons.pool2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>${commons.logging.version}</version>
+        </dependency>
```

Comment:
I think that we only get a splice-derby.log via the slf4j logging functions that our codebase uses. since the zeppelin-splice-interpreter is really just a mini version of our database engine, 'our' logging does (i think) require that logging framework.